be/src/vec/exec/scan/vfile_scanner.cpp

```diff
@@ -46,9 +46,10 @@ VFileScanner::VFileScanner(RuntimeState* state, NewFileScanNode* parent, int64_t
           _profile(profile),
           _strict_mode(false) {}
 
-Status VFileScanner::prepare(VExprContext** vconjunct_ctx_ptr) {
+Status VFileScanner::prepare(VExprContext** vconjunct_ctx_ptr,
+                             std::map<std::string, ColumnValueRangeType>& colname_to_value_range) {
     SCOPED_CONSUME_MEM_TRACKER(_mem_tracker);
-
+    _colname_to_value_range(colname_to_value_range);
```

Comment:
You can just save `_colname_to_value_range` in file scan node and use a pointer in scanner to point to it.
So that we don't need to copy this for every scanner.